français-French.md

```diff
@@ -12,3 +12,99 @@ _Contents:_
 * Use formal "vous" (not informal "tu").
 * For example, <q>You may use this video if you include a link to this page.</q> ([in context](https://www.w3.org/WAI/videos/standards-and-benefits/#permission))<br>-> <q>Vous pouvez utiliser cette vidéo si vous incluez un lien vers la présente page.</q>
 * Background: [GitHub issue: "you" translate formal](https://github.com/w3c/wai-website/issues/123)
+
+
+
+## Linked content name reference
+
+**Web Content Accessibility Guidelines (WCAG) 2.0**
+<br>->Translation: Les Règles pour l'accessibilité des contenus Web (WCAG) 2.0
+
+**W3C technical reports index**
+<br>->Translation: Catalogue des rapports techniques du W3C
+
+**Web Content Accessibility Guidelines Working Group**
+<br>->Translation: Groupe de travail sur les Règles pour l'accessibilité des contenus Web
+
+**Web Accessibility Initiative (WAI)**
+<br>->Translation: l'Initiative pour l'accessibilité du WEB (WAI)
+
+**WCAG Working Group charter**
+<br>->Translation: la charte du groupe de travail sur les WCAG
+
+**WAI Technical Activity**
+<br>->Translation: l'activité technique de la WAI
+
+**Web Content Accessibility Guidelines (WCAG) Overview**
+<br>->Translation: Vue d'ensemble des Règles pour l'accessibilité des contenus Web (WCAG)
+
+**Sufficient and Advisory Techniques in Understanding WCAG 2.0**
+<br>->Translation: Les techniques suffisantes et les techniques recommandées dans Comprendre les WCAG 2.0
+
+**WCAG 2.0 Supporting Documents**
+<br>->Translation: Les documents d'accompagnement des WCAG 2.0
+
+**WAI Resources**
+<br>->Translation: Les ressources de la WAI
+
+**Understanding Programmatically Determined**
```

Comment:
Seems like a specific link to a page. Maybe only use "programmatically determined" in the glossary ?